CSC.md

```diff
@@ -0,0 +1,19 @@
+# 中国学生社区 (Chinese Student Community) 相关内容
```

Comment:
Please use https://raw.githubusercontent.com/mingcns/mingcns.github.io/master/appendix/new_doc_template.md as template for the correct doc gen behavior.